### PR TITLE
Add Modrinth App (fixes #148)

### LINKED ISSUE #148: Modrinth launcher not on install page
Hi !
I just wanted to bring the fact that the Modrinth launcher (that just launched) isn't mentioned here, even if there is quilt as an option to create an instance
image

(Sorry if this issue is not in the right place / Not useful, I am "new" to github) !

### PR DESCRIPTION
**Pull request by Ecorous**
---
See preview on Cloudflare Pages: https://preview-151.quiltmc-org.pages.dev

### CODE REVIEW

**Comment by Ecorous** [comment]
fixes #148

**Comment by Southpaw1496** [comment]
Thank you so much for saving me this job 😅 

Would you be able to rebase with the main branch?

**Review by OroArmor — APPROVED** [review]

**Comment by ix0rai** [comment]
should we advise about the version ordering, or have they fixed it?

File: src/pages/en/install/modrinth.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+permalink: /install/modrinth/
+
+layout: /src/layouts/InstallPage.astro
+platformKey: platform-modrinth
+website: https://modrinth.com/app
+---
+
+import Message from "@atoms/Message.astro";
+
+The Modrinth App is a unique, open source launcher that allows you to play your favorite mods, and keep them up to date, all in one neat little package.
```
Comment (Southpaw1496) [inline src/pages/en/install/modrinth.mdx:12]:
```suggestion
The Modrinth App is a unique, open source launcher that allows you to play your favorite mods, and keep them up to date, all in one neat little package. To set up your own Quilt profile, simply follow these steps:
```
This is more consistent with the other guides.

File: src/pages/en/install/modrinth.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+1. Download and install the Modrinth App from [their webiste](https://modrinth.com/app)
+2. Open Modrinth and click on the `+` in the bottom left
```
Comment (Southpaw1496) [inline src/pages/en/install/modrinth.mdx:15]:
```suggestion
2. Open the app and click on the `+` button in the bottom left
```
Saying "Open Modrinth" could be confused with opening the website.

**Comment by Ecorous** [comment]
> should we advise about the version ordering, or have they fixed it?

That's been fixed

File: src/pages/en/install/modrinth.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+3. Fill out the Name
```
Comment (Southpaw1496) [inline src/pages/en/install/modrinth.mdx:16]:
```suggestion
3. Give the profile a Name
```

File: src/pages/en/install/modrinth.mdx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+4. Pick Quilt from the list of modloaders.
```
Comment (Southpaw1496) [inline src/pages/en/install/modrinth.mdx:17]:
```suggestion
4. Pick Quilt from the list of modloaders
```
The full-stop here is inconsistent with the other bullet points.

**Review by Southpaw1496 — CHANGES_REQUESTED** [review]
Pretty good, just some wording suggestions.

**Comment by Ecorous** [comment]
Should be done now.

**Review by Southpaw1496 — APPROVED** [review]

**Pull request merged**